Taskfile.yml

```diff
@@ -49,6 +49,17 @@ tasks:
           poetry run python -W ignore utils/taskcluster_downloader.py
           --mode=model {{.CLI_ARGS}}
 
+  local-remote-settings:
+    desc: Runs models for use within Firefox via local Remote Settings.
+    summary: |
+      The artifacts will be saved to: ./data/artifacts
+      Example: `task local-remote-settings -- --task-group-ids GU9ZyWFhRDe_nxlAHcen8g`
+    deps: [poetry-install-utils]
```

Comment:
> [!CAUTION] 
> I had to add `zstandard = "^0.22.0"` to the `[tool.poetry.group.utils.dependencies]` list for this to work.
> ```diff
> diff --git a/pyproject.toml b/pyproject.toml
> index 787fa1ab..cfbb3236 100644
> --- a/pyproject.toml
> +++ b/pyproject.toml
> @@ -48,6 +48,7 @@ taskcluster-taskgraph = "^11.1.0"
> kinto-http="11.7.0"
> # Use an outdated version of pydantic due to dependency requirements conflict.
> pydantic="1.10.19"
> +zstandard = "^0.22.0"
>
> # This install group is for running tests. Note that any dependencies in the
>  # pipeline are installed separately through the run_task test abstraction. This
> ```

Reply:
This is actually an issue on all of our utils at the moment. I'd like to follow-up and consolidate the test/utils groups I think.

Reply:
https://github.com/mozilla/translations/issues/971